apps/mark/backend/src/app.ts

```diff
@@ -159,6 +182,64 @@ export function buildApi(
       logger,
       store: workspace.store.getUiStringsStore(),
     }),
+
+    incrementBallotsPrintedCount() {
+      store.setBallotsPrintedCount(store.getBallotsPrintedCount() + 1);
+    },
+
+    async setPollsState(input: { pollsState: PollsState }) {
+      const newPollsState = input.pollsState;
+      const oldPollsState = store.getPollsState();
+
+      store.setPollsState(newPollsState);
+
+      assert(newPollsState !== 'polls_closed_initial');
+      const logEvent = (() => {
+        switch (newPollsState) {
+          case 'polls_closed_final':
+            return LogEventId.PollsClosed;
+          case 'polls_paused':
+            if (oldPollsState === 'polls_closed_final') {
+              // logging case handled by ResetPollsToPausedButton
```

Comment:
Would be ideal, but since it's in `libs/ui` it's difficult to do typescript-wise. Kofi did something like this for `uiStrings`, but that took a while to set up so I don't want to go there in this PR.